migration/freshdesk.rst

```diff
@@ -0,0 +1,274 @@
+from Freshdesk
+**************
+
+Limitations
+===========
+
+Please note below Freshdesk specific limitations. 
+These are additional limitations to the
+:ref:`general ones listed <migration_limitations>`.
+
+   * | Differential migrations are **not** supported!
+     | The general suggestion is to run a test import before to learn
+       how long the migration is going to take.
+   * **Important:** Please note that migration speed highly depends on your
+     Freshdesk plan (API rate limits apply)
+   * Due to API limitations Zammad will not show the total number of objects
+     to import, but instead correct them in steps of ``100``.
+   * User passwords are not migrated and will require password resets.
```

Comment:
How do I reset all passwords? Link or elaboration would be helpful here.

Reply:
You can't. I would never -ever- suggest a default password for all user accounts.
Do you think expecting the users to reset their passwords is a bad behavior?

Passwords that are known to the admin are -per definition- unsafe.

Reply:
I thought more of a function that sends password reset mails to all agents and admins. But as I learned, there is no function like that, I suggest the following sentence:
User passwords are not migrated and will require the users to use the password reset link on the login page.

Reply:
Okay. I also added a link to the security section of the admin documentation as I felt that is absolutely required for context.